HelpSource/Classes/Server.schelp

```diff
@@ -244,6 +249,47 @@ argument:: limit
 The number of times to check for a successful boot.
 argument:: onFailure
 A function to evaluate after the server fails to boot. If onFailure is not given, an error message is posted. Providing a function suppresses the error message. If you want to supply a function and print the normal error message, make sure that your function returns "false," e.g. code::s.doWhenBooted(onFailure: { ... custom action...; false })::.
+argument:: clock
+The default link::Classes/Clock:: is link::Classes/AppClock::. Use link::Classes/SystemClock:: or link::Classes/TempoClock:: if time accuracy is important.
+
+discussion::
+When using the default clock of code::.waitForBoot:: and code::.doWhenBooted::, the pair of braces surrounding the curly bracket can be omitted:
+
+code::
+s.waitForBoot({ ... })
+s.doWhenBooted { ... }
+::
+
+To play sound or music with more precise time control, the fourth argument clock should be set, and then link::Overviews/GUI-Classes##GUI classes:: should be used via code::defer:: as in the form of a function by enclosing it with a pair of curly braces.
+
+code::
+(
+~tempo = TempoClock;
```

Comment:
`tempo` is a misleading variable name. But since this is an example file, why not just write `t = `. Then it doesn't change the current environment.

Reply:
Thanks! Done in https://github.com/supercollider/supercollider/pull/6198/commits/e30d720f58b59009340438afd158a2b8f94f44d3